src/routes/view-space.js

```diff
@@ -31,7 +31,9 @@ const Parent = styled.div`
 function Hours(props) {
     return (<div>
         <h3>Hours</h3>
-        <p>{props.val}</p>
+        {props.val.map((time, i) => {
+            return <p key={i}>{time}</p>
+        })}
     </div>)
```

Comment:
just one comment about this - having each date on a separate <p> makes the section look kind of awkward. if </br> looks better I would suggest that you use it. otherwise it isn't too big a deal

Reply:
> aka, have the `<p>` tag outside of the map and have the map return `{time}</br>`